docs/semgrep-code/getting-started.md

```diff
@@ -2,7 +2,7 @@
 slug: getting-started
```

Comment:
Hey Katie. FYI, I like that your changes to this doc put the focus on Semgrep **Code**. We probably will repeat this convention of Overview + Scan your codebase for all the products in Semgrep.

Just curious - do we assume that the user has gone through the quickstart? If so, should we not have a "If you have not done so already, set up your Semgrep account by following the steps in Quickstart" or something like that?

Reply:
That's a good idea; I'll create a component for this, since I think we'll need it for all the top-level product pages